tests/test_hdf_ioc.py

```diff
@@ -384,6 +391,53 @@ async def test_hdf5_ioc_parameter_validate_works(
     assert val == str(tmp_path) + "/name.h5"  # put should have been stopped
 
 
+@pytest.mark.parametrize(
+    "create_depth, path, expect_exists",
+    [
+        (0, "panda_test1", False),
+        (-2, "panda_test2", True),
+        (-1, "panda_test3/depth_2", False),
+        (1, "panda_test4/depth_2", True),
+        (0, "/", False),
+        (-1, "/panda_test5", False),
+        # make sure our final test passes, so final status message is "OK"
```

Comment:
I'm unsure as to the reason behind this comment; we shouldn't care what the final status message is, every test should be resetting it on each iteration and no bleed-through from one test to another should occur?

Reply:
This was a leftover from me being confused at a test failure. Turns out it was catching a typo with the status message created by the callback, which was `Ok` instead of `OK`. Removed this.